### PR TITLE
completed prep-javascript-variables exercise

### PR DESCRIPTION
**Pull request by Bxvant**
JS variables

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      const numberOfPets = 1;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      const likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      let empty;
+      console.log(empty);
+      console.log(typeof empty);
+      const unset = null;
```
Comment (shawnkost) [inline prep-javascript-variables/index.html]: These should be swapped. `empty` should be getting assigned `null`, while `unset` shouldn't get assigned anything

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just one minor change here!

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (Bxvant) [reply prep-javascript-variables/index.html]: gotcha ! i just changed it.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**